bookwyrm/activitypub/base_activity.py

```diff
@@ -52,10 +52,14 @@ def naive_parse(activity_objects, activity_json, serializer=None):
         if activity_json.get("publicKeyPem"):
             # ugh
             activity_json["type"] = "PublicKey"
+
+        activity_type = activity_json.get("type")
         try:
-            activity_type = activity_json["type"]
             serializer = activity_objects[activity_type]
         except KeyError as e:
+            # we know this exists and that we can't handle it
+            if activity_type in ["Question"]:
```

Comment:
Just mentioning: Could/Should/… this be in an environment variable, so that the list of unsupported features is “documented”?

Reply:
While it _is_ weird to hard code it like this, I thinkputting it in as an environment variable would suggest that changing it there would have a meaningful effect on the how the app functions. If someone running the site changed the environment variable, it would just cause a bunch of unexpected errors.